WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

```diff
@@ -317,28 +317,31 @@ To configure a file server failover cluster, follow the below steps.
    > [!NOTE]
    > If you are using static IP Addresses, you will need to select the network to use and input the IP Address it will use for the cluster name.  If you are using DHCP for your IP Addresses, the IP Address will be configured automatically for you.
 
-6. Choose **Next**.
+9. Choose **Next**.
+
+10. In the **Select Storage** window, select the additional drive (not the witness) that will hold your shares and **Next**.
 
-7. In the **Select Storage** window, select the additional drive (not the witness) that will hold your shares and **Next**.
+11. On the **Confirmation** page, verify your configuration and select **Next**.
 
-8. On the **Confirmation** page, verify your configuration and select **Next**.
+12. On the **Summary** page, it will give you the configuration it has created.  You can select View Report to see the report of the file server role creation.
 
-9. On the **Summary** page, it will give you the configuration it has created.  You can select View Report to see the report of the file server role creation.
+   > [!NOTE]
+   > If the role does not add or start correctly, the CNO (Cluster Name Object) may not have permission to create objects in Active Directory. The File Server role requires a Computer object of the same name as the "Client Access Point" provided in Step 8.
 
-10. Under **Roles** in the console tree, you will see the new role you created listed as the name you created.  With it highlighted, under the **Actions** pane on the right, choose **Add a share**.
+13. Under **Roles** in the console tree, you will see the new role you created listed as the name you created.  With it highlighted, under the **Actions** pane on the right, choose **Add a share**.
 
-11. Run through the share wizard inputting the following:
+14. Run through the share wizard inputting the following:
 
     - Type of share it will be
     - Location/path the folder shared will be
     - The name of the share users will connect to
     - Additional settings such as Access-based enumeration, caching, encryption, etc
```

Comment:
I know that this text is outside of the text you changed, but I would prefer to see a period at the end, since the word "etc" is meant to be an abbreviation of "et cetera" (meaning "and so on").
```suggestion
    - Additional settings such as Access-based enumeration, caching, encryption, etc.
```